File: app/src/main/java/ru/michanic/mymot/Models/SearchFilterConfig.java
package ru.michanic.mymot.Models;

public class SearchFilterConfig {

    Location selectedRegion;
    Manufacturer selectedManufacturer;
    Model selectedModel;
    int priceFrom;
    int priceFor;

    public Location getSelectedRegion() {
        return selectedRegion;
    }

    public void setSelectedRegion(Location selectedRegion) {
        this.selectedRegion = selectedRegion;
    }

    public Manufacturer getSelectedManufacturer() {
        return selectedManufacturer;
    }

    public void setSelectedManufacturer(Manufacturer selectedManufacturer) {
        this.selectedManufacturer = selectedManufacturer;
    }

    public Model getSelectedModel() {
        return selectedModel;
    }

    public void setSelectedModel(Model selectedModel) {
        this.selectedModel = selectedModel;
    }

    public int getPriceFrom() {
        return priceFrom;
    }

    public void setPriceFrom(int priceFrom) {
        this.priceFrom = priceFrom;
    }

    public int getPriceFor() {
        return priceFor;
    }

    public void setPriceFor(int priceFor) {
        this.priceFor = priceFor;
    }
}

File: app/src/main/java/ru/michanic/mymot/Kotlin/Models/SearchFilterConfig.kt
package ru.michanic.mymot.Kotlin.Models

class SearchFilterConfig {
    var selectedRegion: Location? = null
    var selectedManufacturer: Manufacturer? = null
    var selectedModel: Model? = null
    var priceFrom = 0
    var priceFor = 0
    val mainSearchTitle: String?
        get() {
            var sectionTitle: String? = "Все мотоциклы"
            if (selectedRegion != null) {
                sectionTitle = selectedRegion!!.name
            }
            if (priceFrom > 0) {
                sectionTitle += if (priceFor > 0) {
                    ",\n$priceFrom - $priceFor руб."
                } else {
                    ",\nЦена от $priceFrom руб."
                }
            } else if (priceFor > 0) {
                sectionTitle += ",\nЦена до $priceFor руб."
            }
            return sectionTitle
        }
}